runtime/test/test-util.js

```diff
@@ -42,9 +42,11 @@ function assertViewWillChangeTo(setView, entityClass, field, expectations) {
     view.on('change', () => view.toList().then(result => {
       if (result == undefined)
         return;
-      if (result.length == expectations.length &&
-          result.every(a => expectations.indexOf(a[field]) >= 0)) {
-        resolve();
+      if (result.length == expectations.length) {
```

Comment:
thanks for fixing this! i figured this cannot be right, but didn't know how to make it better :)